pages/index.js

```diff
@@ -13,21 +12,22 @@ export default function Landingpage() {
     <>
       <container>
         <Title>Your Travel Journal</Title>
-        <Form
-          onSubmit={handleSubmit}
-          aria-labelledby="Where is your journey going?"
-        >
-          <Label htmlFor="text">City or Country?</Label>
-          <input
-            type="text"
-            required
-            id="text"
-            name="text"
-            placeholder="enter your destination..."
-          />
-          <Button type="submit">Submit</Button>
-          <p>{output}</p>
-        </Form>
+        <fieldset>
+          {" "}
+          <Form onSubmit={handleSubmit}>
+            <legend>Where is your journey going?</legend>
+            <Label htmlFor="text">City or Country?</Label>
+            <input
+              type="text"
+              required
+              id="text"
+              name="text"
+              placeholder="enter your destination..."
+            />
+            <Button type="submit">Submit</Button>
+            <p>{output}</p>
+          </Form>
+        </fieldset>
```

Comment:
The `fieldset` should be part of your `form`, not the other way around.
You use the `fieldset` to group elements within the `form` element.
It could look like this:

```suggestion
        <Form onSubmit={handleSubmit}>
          {" "}
          <fieldset>
            <legend>Where is your journey going?</legend>
            <Label htmlFor="text">City or Country?</Label>
            <input
              type="text"
              required
              id="text"
              name="text"
              placeholder="enter your destination..."
            />
           </fieldset>
           <Button type="submit">Submit</Button>
           <p>{output}</p>
          
        </Form>
```